gdspy/__init__.py

```diff
@@ -257,6 +257,25 @@ def area(self, by_spec=False):
         else:
             return 0.5 * abs(poly_area)
 
+    def get_bounding_box(self):
+        """
+        Returns the bounding box for this cell.
+
+        Returns
+        -------
+        out : Numpy array[2,2] or ``None``
+            Bounding box of this cell [[x_min, y_min], [x_max, y_max]], or
+            ``None`` if the cell is empty.
+        """
+        bb = numpy.array(((1e300, 1e300), (-1e300, -1e300)))
```

Comment:
Start with `if len(self.points) == 0: return None`, then the code follows without the if-clause